modules/imaging_browser/README.md

```diff
@@ -78,7 +78,10 @@ mantis_url - This setting defines a URL for LORIS to include a link to for bug r
   `$LORIS/tools/CouchDB_Import_MRI.php` alongside all other CouchDB
   import scripts, but should logically be considered part of this module.)
 - The imaging browser module includes links to BrainBrowser to visualize data.
+- The control panel on the viewsession page includes links to instruments
```

Comment:
I would tend to right "View Session" page

Reply:
done, changed in in the 4 occurrences